### ISSUE TITLE
Windows build of ice 1.6.0 fails due to CREDITS

### ISSUE THREAD

**Issue by cmb69** [ice/framework#267]
The CREDITS file in ice-1.6.0 has the following contents:
````
Mariusz Laczak [mruz] <mruz@php.net> (lead)
````
This breaks the build on Windows, because the `<` and `>` are not escaped, and as such are treated as redirection signs. It might be argued that the [PHP-SDK](https://github.com/microsoft/php-sdk-binary-tools/) should cater to this escaping, but usually there are no special characters contained in CREDITS.

Furthermore, the first line of CREDITS should contain the name of the package.

So I suggest to change CREDITS to
````
Ice Framework
Mariusz Laczak
````

PS: it's not about the PHP-SDK, but rather the build system of https://github.com/php/php-src/.

**Comment by mruz** [comment]
Thanks! I will change.

**Comment by mruz** [comment]
I have a small patch, so will fix CREDITS in the 1.6.1 release. 
Just wonder how _PEAR_PackageFileManager2_ will update the changelog in the package.xml. I used it for the first version release on pecl. If it doesn't work I will add the changelog manually.
